independent-projects/tools/devtools-testing/src/test/resources/__snapshots__/QuarkusCodestartGenerationTest/generateDefault/pom.xml

```diff
@@ -73,6 +73,25 @@
           </systemPropertyVariables>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>${surefire-plugin.version}</version>
```

Comment:
I don't understand the rationale of this change? Why would you run them always even if the native profile is not enabled.

My guess for the failure is that maybe you are triggering the native build always, and that doesn't work very well on Windows.

I don't have time to check and help though.

Reply:
Not native build - integration tests using `@QuarkusIntegrationTest`. You'd want to run `@QuarkusIntegrationTest` tests against whatever artifact is built ([see the guide](https://quarkus.io/guides/getting-started-testing#quarkus-integration-test)).